src/pydata_sphinx_theme/assets/styles/extensions/_sphinx_design.scss

```diff
@@ -323,7 +323,7 @@ details.sd-dropdown {
     }
 
     // Focus ring
-    &:focus-visible {
+    &:focus:focus-visible {
```

Comment:
This works by increasing the specificity of the selector. If an element is `:focus-visible` then it is also `:focus`, so this changes nothing, it just allows these rules to override the focus styles in Sphinx Design.